src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,145 @@
+package com.airlines.dao;
+
+import com.airlines.entity.Citizenship;
+import com.airlines.entity.CrewMember;
+import com.airlines.entity.Position;
+import com.airlines.exception.DaoOperationException;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class CrewMemberDaoImpl implements CrewMemberDao {
+
+    private final static String INSERT_CREWMEMBER_SQL = "INSERT INTO crew_members(first_name, last_name, position, birthday, citizenship) VALUES (?, ?, ?, ?, ?);";
+    private final static String SELECT_CREWMEMBER_BY_ID_SQL = "SELECT * FROM crew_members WHERE id = ?;";
+    private final static String UPDATE_CREWMEMBER_SQL = "UPDATE crew_members SET first_name =?, last_name = ?, position = ?, birthday = ?, citizenship = ? WHERE id = ?;";
+    private DataSource dataSource;
```

Comment:
public static final 
public static final 

private static final ...
private static final ...
private static final ...
`                                                             <---empty line`
private
private

Class fields are grouped by type - constants (static final first ), by modificator - public, then private and so on:
public
protected
[default]
private

...static final

...final

...private

Reply:
fixed in 45b9bb5fd29cdf8936e2dab8daa2a25ad85a7b59